modules/idlgen/core/src/test/scala/ProtoCodeGenTests.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017-2018 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package freestyle.rpc.idlgen.protobuf
+
+import freestyle.rpc.common.RpcBaseTestSuite
+import java.io.File
+import scala.io.Source
+
+class ProtoCodeGenTests extends RpcBaseTestSuite {
+
+  s"$ProtoCodeGen.generate()" should {
```

Comment:
This begins to address https://github.com/frees-io/sbt-freestyle-protogen/issues/18, more to come.
Currently breaks in Travis due to working directory assumption, will fix

Reply:
Made a temporary workaround for testing in Travis, but using the filesystem will not be sustainable so I'll convert the non-plugin parts to use input streams.